packages/react/src/generators/library/library.ts

```diff
@@ -172,20 +172,12 @@ function addProject(host: Tree, options: NormalizedSchema) {
 
   if (options.publishable || options.buildable) {
     const { libsDir } = getWorkspaceLayout(host);
+    const external = ['react/jsx-runtime'];
 
-    const external = ['react', 'react-dom'];
-    // Also exclude CSS-in-JS packages from build
-    if (
-      options.style !== 'css' &&
-      options.style !== 'scss' &&
-      options.style !== 'styl' &&
-      options.style !== 'less' &&
-      options.style !== 'none'
-    ) {
-      external.push(
-        ...Object.keys(CSS_IN_JS_DEPENDENCIES[options.style].dependencies)
-      );
+    if (options.style === '@emotion/styled') {
+      external.push('@emotion/styled/base');
```

Comment:
Only we need is to modify the import path for `@emotion/styled`.

Reply:
Do we not need `styled-components` as well?  Their docs seem to indicate it is needed. https://styled-components.com/docs/faqs#with-rollupjs

Reply:
@jaysoo Actually, we don't need to add `styled-componetns` because `@nrwl/web:package` generates the Rollup's `external` option from:
- `dependencies`/`devDependencies` from the root `package.json`
- `external` of the project from `workspace.json`
- `dependencies` from `libs/[project]/package.json`

https://github.com/nrwl/nx/blob/bcdbb02aaf7596f92e5351d29496f87f3c6a17fd/packages/web/src/builders/package/package.impl.ts#L255-L258

This means `styled-components` is added automatically as following.

![image](https://user-images.githubusercontent.com/2607019/118295010-51ba3d80-b516-11eb-82bf-e57c87a92d89.png)